CMakeLists.txt

```diff
@@ -1011,18 +1011,36 @@ if(HELICS_ENABLE_PACKAGE_BUILD)
         set(CPACK_RESOURCE_FILE_README "${HELICS_SOURCE_DIR}/README.md")
     endif(WIN32)
 
-    set(CPACK_COMPONENT_APPLICATIONS_DISPLAY_NAME "Application")
+    set(CPACK_COMPONENT_APPLICATIONS_DISPLAY_NAME "Applications")
     set(CPACK_COMPONENT_LIBS_DISPLAY_NAME "Libraries")
     set(CPACK_COMPONENT_HEADERS_DISPLAY_NAME "Headers")
     set(CPACK_COMPONENT_RUNTIME_DISPLAY_NAME "Runtime Libraries")
-
+    set(CPACK_COMPONENT_SWIG_DISPLAY_NAME "SWIG")
```

Comment:
At some point we should split all the CPACK stuff into a separate file, to separate sections a little better.  don't need that this PR, but might be something to consider in the future.